win32/tests/reals/real_lock_free_set.c

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#define GBALLOC_H
+
+#define lock_free_set_create real_lock_free_set_create
```

Comment:
I think we can remove the real_lock_free_set files unless they are being used somewhere